### PR TITLE
feat: load AsyncAPI from **url** query parameter

### LINKED ISSUE #26: Open asyncapi based on URL parameter
#### Reason/Context
Please try answering few of those questions

- Why we need this improvement?
It is useful to be able to call the URL directly and pass in the URL of the async file rather than have to use text box and "Import document" button. For instance if you could use URL https://playground.asyncapi.io/?url=https://raw.githubusercontent.com/asyncapi/asyncapi/master/examples/2.0.0/streetlights.yml to render this async api. Swagger UI and Editor have similar mechanisms.

- How will this change help?
It will allow you to pass around a URL for an async api rather than have to manually specify it in UI.

- What is the motivation? 

#### Description
Please try answering few of those questions
- What changes have to be introduced?
- Will this be a breaking change?
No
- How could it be implemented/designed?

### PR DESCRIPTION
**Pull request by Orodan**
**Description**

- Allow to load async api from 'url' parameter. The behavior is similar to the 'load' parameter.
- Update readme to document use of 'load' and 'url' paramters.

**Related issue(s)**

Resolves #26

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,3 +32,18 @@ docker run -d --name asyncapi-playground -p 83:5000 asyncapi-playground:latest
...
+- Use the *load* parameter.
+
+```
+https://playground.asyncapi.io/?load=https://raw.githubusercontent.com/asyncapi/asyncapi/master/examples/2.0.0/simple.yml
+```
+
+- Use the *url* parameter.
+```
+https://playground.asyncapi.io/?url=https://raw.githubusercontent.com/asyncapi/asyncapi/master/examples/2.0.0/simple.yml
+```
```
Comment (derberg) [inline README.md:48]: How about something like 👇 

## Load AsyncAPI with URL parameter

There are two parameters that serve the same purpose:

- **load** parameter:
   ```
   https://playground.asyncapi.io/?load=https://raw.githubusercontent.com/asyncapi/asyncapi/master/examples/2.0.0/simple.yml
   ```
- **url** parameter:
   ```
   https://playground.asyncapi.io/?url=https://raw.githubusercontent.com/asyncapi/asyncapi/master/examples/2.0.0/simple.yml
   ```

File: src/server/views/app.handlebars [hunk 2]
```diff
@@ -111,6 +109,26 @@
...
+      loadAsyncApi(asyncApiUrl)
+    }
+
+    if (urlParams.get('url')) {
+      const asyncApiUrl = urlParams.get('url')
+
+      updateFetchDocumentUrlElement(asyncApiUrl)
+      loadAsyncApi(asyncApiUrl)
+    }
+
```
Comment (derberg) [inline src/server/views/app.handlebars:131]: I'm always a fan of the approach to have as less code as possible, of course as less as it is still easy to read. What do you think about 👇  code?

```js
    const urlParams = new URLSearchParams(window.location.search);
    const asyncApiUrl = urlParams.get('load') || urlParams.get('url');
    
    if (asyncApiUrl) {
      document.getElementById('fetch-document-url').value = asyncApiUrl;
      loadAsyncApi(asyncApiUrl);
    }
```

**Review by derberg — CHANGES_REQUESTED** [review]
that was quick! thanks

I provided some improvements suggestions, let me know what you think

File: README.md (hunk 1 above)
Comment (Orodan) [reply README.md:48]: Totally ok, no problems :)

File: src/server/views/app.handlebars (hunk 2 above)
Comment (Orodan) [reply src/server/views/app.handlebars:131]: You're right it's more succinct and still very readable 👍

**Comment by derberg** [comment]
@Orodan let me know when I can do another round in review as I can see you did some commits but not all was done yet

**Comment by Orodan** [comment]
@derberg You can check again, thank you for the reviews 😄

File: README.md [hunk 3]
```diff
@@ -32,3 +32,17 @@ docker run -d --name asyncapi-playground -p 83:5000 asyncapi-playground:latest
 
 Then browse to [http://localhost:83/]()
 
+## Load async api
```
Comment (derberg) [inline README.md:35]:
```suggestion
## Load AsyncAPI with URL query parameter
```

**Comment by derberg** [comment]
@Orodan no worries, thanks for your patience. I made one more comment, you know, a change that saves lifes 😄 

btw, if you want to work on another PR, we do not have issue for it yet, but we need improvement in server code to log to the console infomation that the server is started and under what URL the server is available

**Review by derberg — APPROVED** [review]

**Pull request merged**